### PR TITLE
Language select option and clean template from old browsers

### PR DESCRIPTION
**Pull request by azat-io**
Hello!
- Added language select option because your package can be useful by people from another counties.
- Removed trash tags for an old IE browsers. This browsers currently using by less than 1% of people around the world ([proof](http://browserl.ist/?q=%3E+1%25))

Please merge pull request and create new release of your package :smiley:

### CODE REVIEW

File: index.ejs [hunk 1]
```diff
@@ -6,11 +6,7 @@
 <% htmlWebpackPlugin.options.scripts = htmlWebpackPlugin.options.scripts || [] %>
 
 <!DOCTYPE html>
-<!--[if lt IE 7 ]>             <html class="ie6" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 7 ]>                <html class="ie7" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 8 ]>                <html class="ie8" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 9 ]>                <html class="ie9" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <!--<![endif]-->
+<html<% if (htmlWebpackPlugin.files.lang) { %> lang="<%= htmlWebpackPlugin.options.lang %>"<% } %>>
```
Comment (jaketrent) [inline index.ejs]: Good idea.  Please add a default above to 'en'.  Also looks like you removed the manifest support.  Will you add that back, please?

**Comment by azat-io** [comment]
@jaketrent :+1: 
- Back manifest
- Select English as default language

File: index.ejs [hunk 2]
```diff
@@ -6,11 +6,9 @@
...
 
 <!DOCTYPE html>
-<!--[if lt IE 7 ]>             <html class="ie6" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 7 ]>                <html class="ie7" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 8 ]>                <html class="ie8" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 9 ]>                <html class="ie9" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <!--<![endif]-->
+<html
+
+<% if (htmlWebpackPlugin.files.lang) { %> lang="<%= htmlWebpackPlugin.options.lang %>"<% } %><% else { %> lang="en"<% } %><% if (htmlWebpackPlugin.files.manifest)<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>>
```
Comment (jaketrent) [inline index.ejs:11]: @azat-io , I think this default will work, but I wonder if you wouldn't mind using the strategy we've used for defaults thus far instead.  For instance, see line 6:

```
<% htmlWebpackPlugin.options.scripts = htmlWebpackPlugin.options.scripts || [] %>
```

Something like that at the top of the file would be nice.  Then you could avoid the else branch inline.  Sound ok?

**Review by jaketrent — CHANGES_REQUESTED** [review]

**Review by godart — COMMENTED** [review]
I like the idea of dropping this legacy support.

Is there a way this can be merged without the rest (which seems abandoned to me)?

**Comment by TAGC** [comment]
+1 for removing trash tags.

**Comment by rodmoreno** [comment]
What happened to this PR? I would love to have the support to choose the language

**Comment by jaketrent** [comment]
I would like to get this included.  Looks like we have a couple file conflicts at this point.  For some reason, github doesn't know what branch this is on.  The changes are small enough, I could recreate.  @azat-io any input on how you'd like this to proceed, resolving conflicts?  Thanks.

**Comment by jaketrent** [comment]
Reimplemented in 49ea502.

**Comment by jaketrent** [comment]
Published in 5.6.0

**Pull request closed without merging**